<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicPage</class>
 <widget class="QWidget" name="DynamicPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="7">
    <widget class="MessageWidget" name="infoMsg"/>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="ItemView" name="view"/>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="refreshBtn"/>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="addBtn"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="editBtn"/>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="removeBtn"/>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="startBtn"/>
   </item>
   <item row="2" column="6">
    <widget class="QToolButton" name="stopBtn"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemView</class>
   <extends>QTreeView</extends>
   <header>itemview.h</header>
  </customwidget>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QFrame</extends>
   <header>messagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
